src/main/kotlin/lotto/domain/LottoTicket.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class LottoTicket(private val lottoNumbers: Set<LottoNumber>) {
+
+    constructor() : this(LottoNumber.generateNumbers())
+
+    fun getLottoNumbers(): Set<LottoNumber> {
+        return lottoNumbers
+    }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class LottoTicket(val lottoNumbers: Set<LottoNumber>) {

    constructor() : this(LottoNumber.generateNumbers())
```